app/models/product.rb

```diff
@@ -1,12 +1,22 @@
+load 'Validators/validators.rb'
+
 class Product < ApplicationRecord
 
+  PERMALINK_REGEXP =  %r{\A[a-z0-9-]+\Z}.freeze
```

Comment:
capital alphabets are excluded.
can we add some option in regexp?

Reply:
done